jquery.ui.layout/jquery.ui.layout.d.ts

```diff
@@ -2,39 +2,484 @@
 // Project: http://layout.jquery-dev.net/
 // Definitions by: Steve Fenton <https://github.com/Steve-Fenton>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
+// Definitions updated by: Douglas Armstrong <https://github.com/drarmstr>
```

Comment:
please use this format. https://github.com/DefinitelyTyped/definition-header can extract your name.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/jquery/jquery.d.ts#L3